wrangler-transform/src/e2e-test/features/Wrangler/ParseAsCsv.feature

```diff
@@ -0,0 +1,41 @@
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+@Wrangler
+Feature:  Wrangler - Run time scenarios
+
+  @BQ_SOURCE_TEST @BQ_SINK_TEST
+  Scenario: To verify User is able to run a pipeline using parse csv directive
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Plus Green Button to import the pipelines
+    Then Select the file for importing the pipeline for the plugin "Directive_parse_csv"
+    Then Navigate to the properties page of plugin: "BigQueryTable"
+    Then Replace input plugin property: "dataset" with value: "dataset"
```

Comment:
Also set the input plugin property "project" as "projectId", sometimes this test fails locally as it auto picks up the wrong project-id. Might also fix the currently broken build.

Reply:
added.

Reply:
Add the same for BQ sink (Biqquery2) too. The build is failing again because of this.

Reply:
added.